021-type-safety-in-sdks.md

```
@@ -0,0 +1,193 @@
+          "default": null,
+          "encrypt": false
+        }
+      ]
+    }
+  ]
+}
+```
+
+Structure of the typed sdk will be as follows:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)